### ISSUE TITLE
Cannot navigate trough Group chats with Keyboard

### ISSUE THREAD

**Issue by winman3000** [miranda-ng/miranda-ng#2128]
I'm using a Screen Reader. If I open a Group chat window, you cannot tab trough the Controls and you cannot read the messages with the arrow keys as it is with singel chat.

### Steps to reproduce ###

1. Open a group chat.
2. Write something or press TAB to try to navigate to the Message list.
3. Use the arrow keys to navigate trough the message.

Result: it wouldn't work.

**Comment by unitwobble** [comment]
@winman3000 Try pressing Shift+Tab.

**Comment by georgehazan** [comment]
@winman3000 which plugin do you use? tabSRMM, Scriver or StdMsg?

**Comment by winman3000** [comment]
I'm using the StdMsg Plugin.
